### ISSUE TITLE
Avg PSNR and SSIM are relatively low

### ISSUE THREAD

**Issue by PK15946** [mukulkhanna/FHDR#3]
Hi, @mukulkhanna 

I run a test using the provided [2-Iterations model](https://drive.google.com/open?id=13vTGH-GVIWVL79X8NJra0yiguoO1Ox4V) and the 256x256 test set. The result can be seen as follow:
![批注 2020-08-01 091253](https://user-images.githubusercontent.com/34185974/89090742-86ba3100-d3d7-11ea-83b2-ce54475460a5.jpg)

I notice that the avg PSNR and SSIM are not very high. Am I using code in a wrong way? Or it is because the provided pretrain mode is not optimal?

**Comment by mukulkhanna** [comment]
Hi @PK15946, have you had a chance to go through our paper, or any other single exposure HDR reconstruction paper?

16.92 and 0.73 are competitive values for single exposure input HDR reconstruction with the current HDR datasets.

Please refer to the RHS of the quantitative results' table provided in the README [here](https://github.com/mukulkhanna/fhdr#quantitative) for more information.

**Comment by PK15946** [comment]
OK, 16.92 and 0.73 are the results of FHDR/W, how can I get the 20.3 and 0.79? which are the results of FHDR.

**Comment by mukulkhanna** [comment]
@PK15946 You'll need to train the FHDR (4 iterations) model for that.
